core/control/src/svc/CredentialsService.ts

```diff
@@ -23,26 +22,23 @@ export class CredentialsService {
     return CredentialsService.instance;
   }
 
-  async updateGithubConfig(keptnConfig: KeptnGithubCredentials) {
-    const secret = new KeptnConfigSecretFactory().createKeptnConfigSecret(keptnConfig);
-
-    const created = await this.updateGithubCredentials(secret);
-    console.log(created);
-  }
-
   async getKeptnApiToken(): Promise<string> {
     let token;
-    const secret = await this.k8sClient.api.v1
-      .namespaces('keptn').secrets
-      .get({ name: 'keptn-api-token', pretty: true, exact: true, export: true });
-
-    if (secret.body.items && secret.body.items.length > 0) {
-      const apiToken = secret.body.items.find(item => item.metadata.name === 'keptn-api-token');
-      if (apiToken && apiToken.data !== undefined) {
-        token = base64decode(apiToken.data['keptn-api-token']);
-      } else {
-        console.log('[keptn] The secret does not contain the proper information.');
+    try {
+      const secret = await this.k8sClient.api.v1
+        .namespaces('keptn').secrets
+        .get({ name: 'keptn-api-token', pretty: true, exact: true, export: true });
+
+      if (secret.body.items && secret.body.items.length > 0) {
+        const apiToken = secret.body.items.find(item => item.metadata.name === 'keptn-api-token');
+        if (apiToken && apiToken.data !== undefined) {
+          token = base64decode(apiToken.data['keptn-api-token']);
+        } else {
+          console.log('[keptn] The secret does not contain the proper information.');
+        }
       }
+    } catch (e) {
```

Comment:
should we print log this error message some where, or is it clear which error happened? I'd make it more obvious and at least include the name of the error message, if not all details.

Reply:
like in MessageService: Logger.error(e)